config/jobs/kubernetes-sigs/aws-ebs-csi-driver/aws-ebs-csi-driver-presubmits.yaml

```diff
@@ -0,0 +1,40 @@
+presubmits:
+  kubernetes-sigs/aws-ebs-csi-driver:
+  - name: pull-aws-ebs-disk-csi-driver-verify
+    always_run: true
+    decorate: true
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/gcloud-in-go:v20180927-6b4facbe6
```

Comment:
the `gcloud-in-go` image doesn't have runner.sh, just use kubekins?

Reply:
Thx for remind. Updated